buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
+        (let ((i 0) (pub (advertise "destination" "std_msgs/String")))
+            (loop-at-most-every .1
```

Comment:
~~Consider~~ factor~~ing~~ out the whole loop logic. Does lisp also do protocols? Can you write a protocol/interface for looping?